purefa/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,22 @@
+name: PureFA
+files:
+- name: purefa.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/openmetrics
+  - template: instances
+    options:
+    - name: use_openmetrics
+      description: |
+        Use the latest OpenMetrics implementation for more features and better performance.
+        If running on Agent v6, you must set the `python_version` option in `datadog.yaml` to `"3"`.
+      value:
+        example: true
+        type: boolean
+      enabled: true
```

Comment:
You don't need to have `use_openmetrics` config option. This is really only used in integrations that have have multiple different implementations like a legacy version along with an OpenMetrics version.